src/definitions/behaviors/form.js

```diff
@@ -2039,32 +2050,32 @@
             },
 
             minCount: function (value, minCount) {
-                if (minCount == 0) {
+                if (minCount === 0) {
```

Comment:
We should cast minCount to Integer as the minCount value is provided as String from the `get. ancillaryValue` function

Reply:
good catch 👍, I did not done https://github.com/fomantic/Fomantic-UI/pull/2590/commits/5e1720ea963a0e918da79fbc5ce99360edd1f6bb correctly

Reply:
it has a story, I have written a custom eslint rule for it and then did not reviewed the result correctly, thankfully, the problem is/was only in this file and these 6 places you found

and we can fix it nicely, by wrapping the user (maybe string) numbers in `Number(xx)` to cast them for a string comparison

related article: https://thisthat.dev/number-constructor-vs-parse-int/

we will actually support more even /wo `parseInt` - `Number('5.0') === 5`